### PR TITLE
Add benchmark to checkout mutations

### PR DESCRIPTION
**Pull request by fowczarek**
I want to merge this change because of add benchmarks to checkout mutations.

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] Database migration files are up to date.
1. [x] The changes are tested.
1. [x] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for b880386b28bc9ccff1e7b3e1c99c14a60876dcf1 (mirumee/saleor @ Add_more_benchmarks)
Missing base report (7f3659ce81b12e942b1eb88625ce15e1267abcc8). The results couldn't be compared.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark category
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  category view                                     	         23	         23	              0

# api.benchmark checkout mutations
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  add billing address to checkout as anonymous      	         55	         55	             28
  add shipping to checkout as anonymous             	         32	         32	              5
  checkout payment charge as anonymous              	         21	         21	              4
  checkout shipping address update as anonymous     	         67	         67	             35
  checkout voucher code as anonymous                	         62	         62	             34
  complete checkout                                 	         66	         66	             13
  create checkout as anonymous                      	        148	        148	             83
  update checkout lines as anonymous                	         95	         95	             50

# api.benchmark collection
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  collection view                                   	         21	         21	              0

# api.benchmark homepage
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  featured products list                            	         20	         20	              0
  retrieve main menu                                	          8	          8	              0
  retrieve product list                             	          6	          6	              0
  retrieve secondary menu                           	          8	          8	              0
  retrieve shop                                     	          2	          2	              0
  user checkout details                             	         53	         53	             28

# api.benchmark product
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  product details                                   	         17	         17	              3
  retrieve product attributes                       	          9	          9	              0

# api.benchmark variant
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  product variant bulk create                       	         48	         48	              3
  retrieve variant list                             	         23	         23	              9

# api.benchmark variant stocks
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  product variants stocks create                    	         19	         19	              4
  product variants stocks delete                    	         16	         16	              4
  product variants stocks update                    	         23	         23	              4

# api product sorting attributes
  test name                                         	left count 	right count	duplicate count
  --------------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data            	         20	         20	              0
```

</p>
</details>

File: tests/api/benchmark/test_checkout_mutations.py [hunk 1]
```diff
@@ -0,0 +1,991 @@
...
+    api_client,
+    graphql_address_data,
+    stock,
+    product_with_default_variant,
+    product_with_single_variant,
+    product_with_two_variants,
+    count_queries,
+):
+    query = """
+      fragment Price on TaxedMoney {
```
Comment (maarcingebala) [inline tests/api/benchmark/test_checkout_mutations.py]: Don't you want to extract these fragments to some constants and reuse them? For example, this one is defined six times in this module. I would try to extract it and reuse using Python's template strings to construct queries.

Comment (fowczarek) [reply tests/api/benchmark/test_checkout_mutations.py]: It's a copy of a query from the storefront. 
When some part of storefront query will change it's easier to replace the whole query instead od looking for diffs. 
I can split it to parts if you want?

Comment (fowczarek) [reply tests/api/benchmark/test_checkout_mutations.py]: After a small discussion, we decided, to move fragment to const.
In the feature, we should create fragments with all field of type and reuse it as many time as possible.

Comment (maarcingebala) [reply tests/api/benchmark/test_checkout_mutations.py]: Some fragments like addresses or prices could be extracted anyway. I would do that now in this file and later we'll see if we can reuse that in other files as well (but that's a separate PR).

**Pull request merged**